autoware_lanelet2_extension/docs/lanelet2_format_extension.md

```diff
@@ -10,15 +10,42 @@ Users may use `autoware_lanelet2_validation` [node](../README.md#nodes) to check
 The following is the extra format added for Autoware:
 
 - [extra regulatory elements](extra_regulatory_elements.md)
+  - Right Of Way
   - Detection Area
   - Road Marking
-  - Bus Stop Area
+  - Speed Bump
+  - Crosswalk
+  - No Stopping Area
+  - No Parking Area
+  - Bus Stoop Area
```

Comment:
I'm okay with the current order, but do you have any reasons why you put them in this order? Should we rearrange them in alphabetical order?